crates/stc_ts_file_analyzer/src/analyzer/types/mod.rs

```diff
@@ -675,15 +675,20 @@ impl Analyzer<'_, '_> {
         let is_str = types.iter().any(|ty| ty.is_str());
```

Comment:
We can apply `expand_enum_variant` before iterating over types.

https://github.com/dudykr/stc/blob/43dd6021b398b1e29f0884df2e69edf48192ef3b/crates/stc_ts_file_analyzer/src/analyzer/enums.rs#L565-L605

Reply:
If you do so, enum variants will be converted to numeric literals or string literals.

Reply:
oh, thanks I'll apply it.